Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "JuliaInterpreter"
 uuid = "aa1ae85d-cabe-5617-a682-6adf51b2e16a"
-version = "0.9.37"
+version = "0.10.0"
```

Comment:
Doesn't have to be a breaking release?

Reply:
As far as I recall, bumping the supported Julia version typically involves at least a minor version bump (based on my experience with Cthulhu.jl). But is that not always the case?